target/linux/ramips/dts/mt7621_tplink_archer-x6-v3.dtsi

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
@blocktrron Looks like this one has been relicensed compared to the source of the copy-paste ...?

Reply:
> @blocktrron Looks like this one has been relicensed compared to the source of the copy-paste ...?

@adschm, thanks for pointing out,  this was perhaps introduced when I had issues with squashing multiple comment (due to a deleted commit on which my tree was branched) . The license should not be changed.